src/main/java/com/opentable/metrics/MetricSetBuilder.java

```diff
@@ -45,16 +45,22 @@
 
     private MetricSet built;
 
+    /**
+     * Create a Metric Set Builder
+     */
     public MetricSetBuilder() { }
 
     /**
+     * Create a Metric Set Builder
      * @param registry the registry to register metrics on
      */
     public MetricSetBuilder(MetricRegistry registry) {
         this.registry = registry;
     }
 
     /**
+     * Set the metric registry. If set, the metric set will be registered on this metric set when built,
```

Comment:
I was unable to decode this sentence.  Should it be "registered on this metric registry..."?